### ISSUE TITLE
Please don't waste time

### ISSUE THREAD

**Issue by BlackHole1** [andrewrk/node-s3-client#223]
This library has not been maintained for a long time, and there are a lot of bugs. I have been wasting this library for many hours.

Currently I have encountered bugs: upload can not call end, upload error SigV4

Although it can be solved by partial hacking, this is not a good idea. Will make your code difficult to maintain.

You can use the [@auth0/s3](https://www.npmjs.com/package/@auth0/s3) library, which is a fixed version of s3, and the apis are the same. It runs fine on me this

Author's description: https://github.com/andrewrk/node-s3-client/pull/212#issuecomment-451028324

**Comment by strogonoff** [comment]
If everyone agrees that @auto0/s3 is the main upstream version now, perhaps we should suggest @auth0 to enable issue reporting for that repository?

**Comment by MickL** [comment]
Yes `auth0/node-s3-client` has no issue tracker. Also the dependencies (e.g. aws-sdk) are not up to date, last commit is 2 years old. If `auth0/node-s3-client` is not maintained and has no issue tracker, may someone fork `auth0/node-s3-client`, update package.json and open up the issue tracker?

**Comment by alexanderankin** [comment]
thanks for making this issue
